sql-cloudant/README.md

```diff
@@ -62,6 +62,8 @@ cloudant.protocol|https|protocol to use to transfer data: http or https
 cloudant.host||cloudant host url
 cloudant.username||cloudant userid
 cloudant.password||cloudant password
+cloudant.useQuery|false|When enabled, for query not using index or view, _find will be used instead of _all_docs, some query predicates will be driven into datastore. However, RDD partition is ONE during _find, so parallel loading is not achieved
```

Comment:
I chose the word useQuery, as Query is Cloudant's special word.  Also when useQuery is not used, pushing down can also happens for primary key (_id) query. So the above will not be accurate.